tests/Integration/Bridge/TolerantParser/SourceCodeFilesystem/ScfClassCompletorTest.php

```diff
@@ -70,6 +177,31 @@ public function provideComplete(): Generator
                     'type' => Suggestion::TYPE_CLASS,
                     'name' => 'Clapping',
                     'short_description' => 'Test\Name\Clapping',
+                    'class_import' => 'Test\Name\Clapping',
+                ],
+            ],
+        ];
+
+        yield 'extends keyword with subsequent code, with class already imported' => [
+            '<?php use Test\Name\Clapping; class Foobar extends Cl<> { }',
+            [
+                [
+                    'type' => Suggestion::TYPE_CLASS,
+                    'name' => 'Clapping',
+                    'short_description' => 'Test\Name\Clapping',
+                    'class_import' => null,
+                ],
+            ],
+        ];
+
+        yield 'extends keyword with subsequent code, with class from same namespace' => [
```

Comment:
I think this might be too much -- the tolerant chain completor truncates after the cursor -- so it doesn't matter what comes after it, and this is tested elsewhere